### PR TITLE
Update wstream.py

### PR DESCRIPTION
**Pull request by soter80**
wstream not resolve fix pattern

### CODE REVIEW

**Review by Gujal00 — CHANGES_REQUESTED** [review]
the pattern is not handling the space after `sources:`
also unnecessary escape for " as the ''' covers it already
Also both patterns are being used, so it would be better to have both rather than replacing

File: lib/resolveurl/plugins/wstream.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ class WstreamResolver(ResolveGeneric):
 
     def get_media_url(self, host, media_id):
         return helpers.get_media_url(self.get_url(host, media_id),
-                                     patterns=[r'''sources:\s*\[{file:\s*"(?P<url>[^"]+)'''],
+                                     patterns=[r'''sources:\[\"(?P<url>[^\"]+)\"\]'''],
```
Comment (Gujal00) [inline lib/resolveurl/plugins/wstream.py:29]:
```
patterns=[r'''sources:\s*\[{file:\s*"(?P<url>[^"]+)''',
          r'''sources:\s*\["(?P<url>[^"]+)'''],
```
would be better

Comment (soter80) [reply lib/resolveurl/plugins/wstream.py:29]: But with my change wstream work with r'''sources:\s* \["(?P<url>[^"]+)'''], not work

**Comment by Gujal00** [comment]
I will close this for now, as I have pushed the change to git. 
Good First PR, may be next time I will merge as is 👍

File: lib/resolveurl/plugins/wstream.py (hunk 1 above)
Comment (soter80) [reply lib/resolveurl/plugins/wstream.py:29]: But whit this patterns=[r'''sources:\s*\[{file:\s*"(?P<url>[^"]+) wstream not work with this patterns=[r'''sources:\[\"(?P<url>[^\"]+) work better

Comment (Gujal00) [reply lib/resolveurl/plugins/wstream.py:29]: both patterns are there in the change and work correctly now. Check commit https://github.com/jsergio123/script.module.resolveurl/commit/0eebd297571f95ecb2d01f78e91e3e0f38eb2520

**Pull request closed without merging**